### ISSUE TITLE
Update Quick Tutorial to use pytest instead of nose

### ISSUE THREAD

**Issue by stevepiercy** [Pylons/pyramid#2495]
http://docs.pylonsproject.org/projects/pyramid/en/latest/quick_tutorial/requirements.html uses the following packages:

```text
nose
webtest
deform
sqlalchemy
pyramid_chameleon
pyramid_debugtoolbar
waitress
pyramid_tm
zope.sqlalchemy
```

Ensure that all these packages are needed throughout the Quick Tutorial

**Comment by stevepiercy** [comment]
I'm about 2/3 done https://github.com/stevepiercy/pyramid/tree/docs/quick-tour-gotcher-nose should finish this weekend, so added to 1.7 milestone.

**Comment by stevepiercy** [comment]
closed by #2505
